package.json

```diff
@@ -22,9 +22,9 @@
     "doc-fence-check": "node tools/check-fenced-code.mjs",
     "lint-documentation": "jest --coverage false test/docs/**.spec.ts",
     "lint-other": "jest --coverage false test/other/**.spec.ts",
-    "eslint": "eslint . --cache --cache-location .cache/eslint --report-unused-disable-directives",
-    "eslint-fix": "eslint --cache --cache-location .cache/eslint --fix . --report-unused-disable-directives",
-    "eslint-ci": "eslint . --cache --cache-strategy content --cache-location .cache/eslint --format gha  --report-unused-disable-directives",
```

Comment:
Why drop this reporting? We will accumulate them as a result. Or it's preferred to have them as a warning?

Reply:
it's now inside the config file

Reply:
![image](https://github.com/user-attachments/assets/73a0ad1d-e43e-41a3-98d9-b9b6a73f9ef9)
☝️